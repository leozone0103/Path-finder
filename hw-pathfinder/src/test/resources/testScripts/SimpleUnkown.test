# Find path with an unknown points
CreateGraph graph
AddNode graph A
FindPath graph B A
FindPath graph A B