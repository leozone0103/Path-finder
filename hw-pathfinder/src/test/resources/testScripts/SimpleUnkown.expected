# Find path with an unknown points
created graph graph
added node A to graph
unknown: B
unknown: B